css/standard.css

```diff
@@ -84,11 +84,7 @@ padding: 0px;
 		color: white;
 	}
 	
-	.anim {
-		-o-transition:.5s;
-  -ms-transition:.5s;
-  -moz-transition:.5s;
-  -webkit-transition:.5s;
+.anim {
```

Comment:
transition property has been supported by all major browsers for 

>3 versions of Chrome, Firefox, Safari, Opera
>1 version of IE.

As such, it is recommended not to use prefixes.